src/server/src/Server.cpp

```diff
@@ -77,22 +69,24 @@ bool Server::configure(ResourceFinder& rf)
         return false;
     }
 
-    if (!drdIsInitialized () && drdAutoInit () < 0)
+    if (!drdIsInitialized() && drdAutoInit() < 0)
     {
         std::cout << "Error: cannot initialize the Omega.3 robot." << std::endl;
 
         return false;
     }
-    else if (drdStart () < 0)
+    else if (drdStart() < 0)
     {
         std::cout << "Error: cannot start the Omega.3 robot." << std::endl;
 
         return false;
     }
 
     /* Move in a safe position and idle the robot. */
+
     enable_position_control();
-    drdMoveToPos(0.0, 0.0, 0.0, false);
+    drdMoveToPos(0.0, 0.0, 0.0, false); // stay at initial position
+
```

Comment:
This space wasn't there before!

Reply:
no space anymore

Reply:
Thank you. However, you changed the y coordinate from 0.0 to -1.0.